CMakeLists.txt

```diff
@@ -258,21 +258,21 @@ endif()
 if(CORENRN_ENABLE_NMODL)
   find_package(nmodl)
   if(nmodl_FOUND)
-    set(CORENRN_NMODL_BINARY ${nmodl_BINARY})
-    set(CORENRN_NMODL_INCLUDE ${nmodl_INCLUDE})
+    set(CORENRN_BACKEND_BINARY ${nmodl_BINARY})
+    set(CORENRN_BACKEND_INCLUDE ${nmodl_INCLUDE})
```

Comment:
I completely agree that `CORENRN_NMODL_BINARY` is misleading!  But `CORENRN_BACKEND` is also not clear / intuitive. It's backend for translating MOD file to CPP. So, should we say `CORENRN_MODTOCPP_BACKEND_BINARY` or `CORENRN_MODTOCPP_BINARY`?